### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by du-cki**

### CODE REVIEW

**Comment by du-cki** [comment]
There is a known bug for when a user is non-existent in the database, It'll raise the error in certain areas. I think you can write an `SELECT OR INSERT` SQL statement so we can get rid of the `Database.check_user` function but I'm too tired to think for today. Good night.

**Comment by Kagetasu** [comment]
Looks great. I just need my parentheses back 😄

**Comment by du-cki** [comment]
> Looks great. I just need my parentheses back 😄
> 
> Although we can compromise and include spaces between the if's and the parentheses 

True that, I will check to see if there's a `black` config that can toggle the parentheses back on and make sure to commit it.

File: utils/economy.py
Comment (Kagetasu) [inline utils/economy.py:1]: In functions `update` and `get_balance`, the user's balance is referred to as float. It is meant to be an integer, as defined in the table creation

Comment (Kagetasu) [reply utils/economy.py:1]: `Optional` imported but not used

**Comment by Kagetasu** [comment]
> There is a known bug for when a user is non-existent in the database, It'll raise the error in certain areas. I think you can write an `SELECT OR INSERT` SQL statement in each of the database functions so we can get rid of the `Database.check_user` function but I'm too tired to think for today. Good night.

I'll get to working on that. Great job

File: commands/economy/highlow.py
Comment (Kagetasu) [inline commands/economy/highlow.py:1]: imported (from) discord twice, `discord.Interaction` could be imported `from discord` instead

File: commands/economy/work.py
Comment (Kagetasu) [inline commands/economy/work.py:1]: line 27, `color` of the embed was removed although it is not the same as the default for `Embed` (very minor issue)

**Pull request merged**